crypto/test/datatypes_driver.c

```diff
@@ -139,12 +139,12 @@ int main(void)
 
 void byte_order(void)
 {
-    int i;
+    size_t i;
     v128_t e;
 
     printf("byte ordering of crypto/math datatypes:\n");
     for (i = 0; i < sizeof(e); i++)
-        e.v8[i] = i;
+        e.v8[i] = (uint8_t)i;
```

Comment:
Since `i` is defined right there in the loop, why not make it the needed type to avoid the cast?